packages/compass-e2e-tests/helpers/commands/index.js

```diff
@@ -0,0 +1,17 @@
+// @ts-check
+
+/**
+ * @param {ExtendedApplication} app
+ */
+exports.addCommands = function(app) {
```

Comment:
Split them into one command per file as that will probably scale best. Not sure how best to register them. This works, but maybe there are better ways.

Reply:
I'm not sure why a single file for commands is not scalable tbh (this problem of manually registering them will not exist with it, what are the downsides in your opinion?), but you probably can do something like `fs.readdir(__dirname)` and then autoregister everything that will be found in `./commands` folder deriving the command name from a file name or something if you are looking for a way to remove the need for a manual registration. Not that big of a deal either way

Reply:
In my experience with a few e2e test suites is that most of the code ended up in custom commands. ie. by the time you have a dozen or so e2e tests (which we'll have probably this sprint already) there are way more lines of code in shared commands than in tests and you quickly have a multi-thousand line function to create them. It already felt a bit unwieldy to me.